### PR TITLE
Add option to skip attendances

### PR DESCRIPTION
**Pull request by simahawk**
This was a quick workaround I coded before Thierry came w/ his fix on #10.
I kept because it can be useful to speed up pushing TS. 
Attendances can be pushed once in a week for instance or never if you are an external resource (eg: @i-vyshnevska and myself in a bunch of days :wink:)

### CODE REVIEW

**Review by Tonow-c2c — APPROVED** [review]

**Review by i-vyshnevska — CHANGES_REQUESTED** [review]
you also need to disable login to odoo if option is given

**Comment by simahawk** [comment]
> you also need to disable login to odoo if option is given

you are right. I'll have a look tomorrow

**Comment by simahawk** [comment]
@i-vyshnevska should be better now:

* no login
* no report
* support rc config (no need to pass the flag everytime)

File: README.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ The year will be computed automatically based on current year.
 For instance: if current week is the 1st of 2020 and you push -w 2
 it will push week 51 of 2019.
 
+If you want to skip attendances youn can use `--no-attendance` or set `no_attendance = 1` in `gtimelogrc`.
```
Comment (asaunier) [inline README.md]:
```suggestion
If you want to skip attendances you can use `--no-attendance` or set `no_attendance = 1` in `gtimelogrc`.
```

**Review by i-vyshnevska — APPROVED** [review]
functional test +

**Pull request merged**